src/v2/view-builder/views/ov/ResendView.js

```diff
@@ -4,7 +4,7 @@ import { SHOW_RESEND_TIMEOUT } from '../../utils/Constants';
 
 export default View.extend({
   //only show after certain threshold of polling
-  className: 'hide resend-ov-link-view',
+  className: 'hide resend-ov-link-view margin-top-30',
```

Comment:
How does moving it within form container help ? We would still need the margin to be applied

Reply:
I am going to apply the margin via css

Reply:
@nikhilvenkatraman-okta  the problem here is that when just resend callout is to be displayed it will always have an extra margin top of 30px regardless of an error callout already being present or not. Can you check how it looks. 

I am wondering how it would be if we just move the resend to form 'o-form-fieldset-container' instead of error container.

Reply:
@magizh-okta We can add the margin conditionally when the error callout and resend view are present. It should not add the margin if only resend-ov-link-view  is present.

```
  .o-form-error-container > .infobox-error + .resend-ov-link-view {
    // if error callout and warning callout show up on ov enroll screen, add margin between callouts
    margin-top: 20px;
  }

```
![Screen Shot 2021-03-22 at 1 39 45 PM](https://user-images.githubusercontent.com/23267876/112055442-156cf100-8b14-11eb-822d-ddd5e4b15f2c.png)